invoice/forms.py

```diff
@@ -32,7 +32,7 @@ class CustomerForm(ModelForm):
 
     class Meta:
         model = Customer
-        fields = ['first_name', 'last_name', 'email']
+        fields = ['first_name', 'last_name', 'email', 'vendor']
```

Comment:
This will actually display all vendors. However, we want to limit it to the vendors of the user.

Reply:
Fixed by queryset overwrite in views.